WalletWasabi/BitcoinCore/P2pNode.cs

```diff
@@ -184,7 +185,7 @@ public void Disconnect()
 				{
 					try
 					{
-						Node?.Dispose();
+						node.Dispose();
```

Comment:
Dispose should never throw any exceptions. That is why we are not handling that. You can remove the try/c here.

Reply:
Don't. It's legacy code in NBitcoin code, we cannot risk it on best practice.